lib/membrane_rtc_engine/endpoints/webrtc/track_receiver.ex

```diff
@@ -190,20 +190,13 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackReceiver do
     super(msg, ctx, state)
   end
 
-  # although `:input` pad is static it might not exist
-  #
-  # consider scenario when track receiver is encapsulated
-  # into bin with dynamic input pad
-  #
-  # after unlinking bin's dynamic input pad
-  # we are loosing our static pad
-  defp maybe_request_keyframe(nil, _ctx), do: []
+  defp maybe_request_keyframe(nil), do: []
 
-  defp maybe_request_keyframe(_current_variant, %{pads: %{input: _pad_data}}),
+  defp maybe_request_keyframe(_current_variant),
     do: [event: {:input, %Membrane.KeyframeRequestEvent{}}]
 
-  defp maybe_request_track_variant(nil, _ctx), do: []
+  defp maybe_request_track_variant(nil), do: []
 
-  defp maybe_request_track_variant(variant, %{pads: %{input: _pad_data}}),
+  defp maybe_request_track_variant(variant),
     do: [event: {:input, %RequestTrackVariant{variant: variant}}]
```

Comment:
After implementing removal of TR in handle_pad_removed in WebRTC Endpoint, these checkes are no longer needed